Common/src/main/java/com/blamejared/crafttweaker/api/zencode/scriptrun/IBepRegistrationHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.blamejared.crafttweaker.api.zencode.scriptrun;
+
+import com.blamejared.crafttweaker.api.zencode.bracket.ValidatedEscapableBracketParser;
+import org.openzen.zenscript.parser.BracketExpressionParser;
+
+@FunctionalInterface
+public interface IBepRegistrationHandler {
+    
+    void registerBracketHandler(final String bepName, final BracketExpressionParser parser);
+    
+    // TODO("Public API?")
+    default void registerBracketHandler(final ValidatedEscapableBracketParser parser) {
```

Comment:
Who calls this method and is it something that belongs here, or something where the caller could directly call the other method?